__init__.py

```diff
@@ -68,18 +68,17 @@ async def async_setup_entry(hass, config_entry) -> bool:
         UNDO_UPDATE_LISTENER: undo_listener,
     }
 
-    for component in PLATFORMS:
-        hass.async_create_task(
-            hass.config_entries.async_forward_entry_setup(config_entry, component)
-        )
+    await hass.config_entries.async_forward_entry_setups(
+        config_entry,
+        PLATFORMS
+    )
 
     return True
 
 
 async def async_unload_entry(hass, config_entry):
     """Unload an FMI config entry."""
-    for component in PLATFORMS:
-        await hass.config_entries.async_forward_entry_unload(config_entry, component)
+    await hass.config_entries.async_unload_platforms(config_entry, PLATFORMS)
```

Comment:
This doesn't fix any warning right now, but https://github.com/home-assistant/core/blob/9723b97f4bd42c850f5cb5fbf67ecdfc55769861/homeassistant/config_entries.py#L2175 says that `async_unload_platforms` is the preferred method nowadays.

<img width="497" alt="bild" src="https://github.com/anand-p-r/fmi-hass-custom/assets/1417619/31ad145b-b3e6-4f08-be96-2147e0193cc6">